### DISCUSSION TITLE
Question about reading VCF file

### DISCUSSION THREAD

**Discussion by shy218** [sgkit-dev/sgkit#538] [General]
I currently test the packages on a small vcf dataset, but the chromosome in the vcf file has Y, (I think it is a chromosome for sex?). However, sgkit function vcf_to_zarr() gives an error that "Y is not in list." Do I need to first convert Y into a number?

**Comment by hammer** [comment]
@shy218 I’m glad to hear you’re working with data! Could you please use the issue tracker for issues you encounter when using the software? They’re easier to track that way.

**Reply by hammer** [reply]
Discussions are great for more general topics about what to work on and how things work.

**Reply by shy218** [reply]
I am sorry. I am new to these functions on Github.

**Reply by hammer** [reply]
No worries!
